vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -63,4 +92,35 @@ public ExhibitionAboutPage getExhibitionAboutPage() {
         return aboutPageList != null && !aboutPageList.isEmpty() ? aboutPageList.get(0):new ExhibitionAboutPage();
     }
     
-}
+    /**
+     * This method maps the title in ExhibitionAboutPage, and add that to
+     * exhibitionTitles list for each user selected Exhibition Language.
+    */    
+    public void setAboutPageTitle(LocalizedTextForm title, ExhibitionAboutPage exhibitionAboutPage) {
+        setLocalizedText(title, exhibitionAboutPage.getExhibitionTitles());
+    }
+    
+    /**
+     * This method maps the description in ExhibitionAboutPage, and add that to
+     * exhibitionTextDescriptions list for each user selected Exhibition Language.
+    */
+    public void setAboutPageDescription(LocalizedTextForm aboutPageText, ExhibitionAboutPage exhibitionAboutPage) {
+        setLocalizedText(aboutPageText, exhibitionAboutPage.getExhibitionTextDescriptions());
+    }
+
+    private void setLocalizedText(LocalizedTextForm textForm, List<ILocalizedText> localizedTextList) {
+        if (textForm != null && !StringUtils.isEmpty(textForm.getText())) {
+            LocalizedText localizedText = localizedTextRepo.findById(textForm.getLocalisedTextId()).orElse(null);
+            if (localizedText != null) {
+                localizedText.setText(textForm.getText());
+            } else {
+                ExhibitionLanguage exhibitionLanguage = exhibitionLanguageRepository.findById(textForm.getExhibitionLanguageId()).orElse(null);
+                if (exhibitionLanguage != null) {
+                    localizedText = new LocalizedText(exhibitionLanguage, textForm.getText());
+                    localizedTextList.add(localizedText);
```

Comment:
actually all of the code above seems to be repeated multiple times. Should be more DRY.